cmd/registry/patch/artifact_unknown.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2022 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package patch
+
+import (
+	"github.com/apigee/registry/rpc"
+	"github.com/apigee/registry/server/registry/names"
+	"google.golang.org/protobuf/proto"
+)
+
+const UnknownArtifactMimeType = "application/octet-stream"
+
+type UnknownArtifact struct {
+	Header `yaml:",inline"`
+	Data   struct {
+		MimeType string `yaml:"mimeType,omitempty"`
+	} `yaml:"data"`
+}
+
+func (a *UnknownArtifact) GetMimeType() string {
+	return UnknownArtifactMimeType
```

Comment:
If we're parsing the MIME type from the YAML why doesn't this return that value?

Reply:
Done.